src/redux/books/books.js

```diff
@@ -0,0 +1,34 @@
+// Constants
+const ADD_BOOK = 'bookstore/src/redux/books/addBook';
+const REMOVE_BOOK = 'bookstore/src/redux/books/removeBOOK';
+
+// Action creators
+/* const initialBook = [
+  { id: 0, title: 'Operating Systems', author: 'xxx' },
+  { id: 1, title: 'Computer Architecture', author: 'william Stallings' },
+  { id: 2, title: 'Software Engineering', author: 'Robert Mitchil' },
+]; */
```

Comment:
- [ ] Great work implementing the previously required changes, your app works fine. However, due to the refactor you have left commented code in your application. Please remove the commented code for a cleaner codebase and a more professional looking one.

```
components folder files to consider
- book.js
- InputBook.js
```

```
redux folder files to consider
- books.js
- configureStore.js
```